{"name": "При указании неверного пароля зарегистрированного пользователя, в ответе возвращается ошибка 401: \"email or password are incorrect\"", "status": "passed", "steps": [{"name": "Авторизовать пользователя", "status": "passed", "parameters": [{"name": "user_data", "value": "{'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY1YmNhMmFjOWVkMjgwMDAxYjNiMGY0YyIsImlhdCI6MTcwNjg2MTIyOCwiZXhwIjoxNzA2ODYyNDI4fQ.GMgwYuX71UW4seaIdJRDEr8qYUY_pwqaBCwgj_ucowI', 'email': 'jason07@example.com_wrong', 'password': '^moQRwdy67', 'name': 'Andre'}"}], "start": 1706861226733, "stop": 1706861226906}], "start": 1706861226731, "stop": 1706861226910, "uuid": "e53605d4-a153-4d1f-bbb0-669ee1cfba44", "historyId": "27d7b0cd974dec96b0423bfafdb403c6", "testCaseId": "27d7b0cd974dec96b0423bfafdb403c6", "fullName": "tests.test_login_user.TestLoginUser#test__login_user__user_data_wrong_password__incorrect", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "Lenovo-1"}, {"name": "thread", "value": "11884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}